Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -5,24 +5,214 @@
 import Foundation
 import UIKit
 
-private let StatusBarHeight = 20
+private let StatusBarHeight = CGFloat(20)
+private let TextBoxHeight = CGFloat(25.0)
+private let Margin = CGFloat(10)
+private let ToolbarHeight: CGFloat = 44
+private let CellHeight = TextBoxHeight * 4
+
+@objc
+class TransitionOptions {
```

Comment:
BrowserViewController uses that cell (all the table rows are CustomCell objects) in its transition It keeps the urlbar lined up with the view (and fading out) as it moves so that the two look connected. Unfortunately, because its behind the TabsTray, it slides behind it and disappears rather quickly. Even if we move the browser in front of the tabs tray, I'd like some way to pass information back and forth between the two view controllers for the transition. i.e. we'd want to tell the BrowserViewController ("the webview should overlap this frame")

We could call the options list namedViews or something, since essentially its used to find views that should have matching positions before/after the transition. That's more restrictive, which I don't love here.